### PR TITLE
Capitalize Rmd file extension for consistency

### PR DESCRIPTION
**Pull request by prosoitos**

### CODE REVIEW

**Comment by prosoitos** [comment]
I added 2 line breaks, in accordance with 

> Use a separate line for the target of the assignment followed by <-  

from http://style.tidyverse.org/pipes.html#assignment-1.

I could not respect your standard 2 spaces indent as I work in emacs and the package I use to auto-indent in ESS is set to 4 spaces. Sorry about that. I might be able to change that setting, or I could correct that directly on GitHub.

**Comment by prosoitos** [comment]
Indent sizes fixed.

**Comment by prosoitos** [comment]
Fixed 2 small typos (missing words) and one grammar issue

File: iteration.Rmd [hunk 1]
```diff
@@ -846,7 +846,8 @@ x %>%
 
 ```{r, eval = FALSE}
 library(ggplot2)
-plots <- mtcars %>% 
+plots <-
```
Comment (hadley) [inline iteration.Rmd]: Can you leave this as is for now?

Comment (prosoitos) [reply iteration.Rmd]: Sorry I missed this comment for so long...
Just reverted that change.

**Comment by hadley** [comment]
Sorry for not merging this, but it looks like we've now consistently switch to `.Rmd`

**Pull request closed without merging**